### ISSUE TITLE
Loading Multiple .Less Files In Parallel Fails. NuGet v1.5.1

### ISSUE THREAD

**Issue by duncanmcdougall** [dotless/dotless#493]
<head>
       <link rel="stylesheet" href="/Content/app.less" media="screen"/>
       <link rel="stylesheet" href="/Content/print.less" media="print"/>
    </head>

One of these will return a 500 error if both files generated CSS hasn't been cached by dotless yet. I suspect this is due to the CSS being generated. When caching is enabled, on second load of the page both files will return the expected CSS.

### How to replicate ###
 
From package manager console in VS. 

    Install-Package dotless
    Install-Package Twitter.Bootstrap.Less

Set your dotless configuration to the following

    <dotless minifyCss="false" cache="true" web="true" strictMath="false" />

Create three .less files in /Content - test1.less, test2.less, test3.less and set the following as the content. 

    @import "bootstrap/bootstrap.less";

Copy this code to your _Layout.cshtml

    <!DOCTYPE html>
    <html lang="en">
    <head>
        <title></title>
        <meta http-equiv="content-type" content="text/html;charset=UTF-8"/>
        <link rel="stylesheet" href="/Content/test1.less" media="screen"/>
        <link rel="stylesheet" href="/Content/test2.less" media="screen"/>
        <link rel="stylesheet" href="/Content/test3.less" media="screen"/>
    </head>
    <body>
        @RenderBody()
    </body>
    </html>

#### Expected Result ####
1 or 2 of the files will return a 500 error.

**Comment by rytmis** [comment]
Can you please see if this reproduces with 1.5.2-beta2? This sounds like it could be a duplicate of #480.

**Comment by rytmis** [comment]
I'm going with the assumption that 1.5.2 fixes this. If that's not the case, then please reopen the issue.
